packages/perseus/src/widget-ai-utils/categorizer/prompt-utils.ts

```diff
@@ -0,0 +1,33 @@
+import type {WidgetType} from "../../prompt-types";
+import type {PerseusCategorizerUserInput} from "../../validation.types";
+import type categorizer from "../../widgets/categorizer/categorizer";
+import type {PropsFor} from "@khanacademy/wonder-blocks-core";
+
+type WidgetProps = PropsFor<typeof categorizer.widget>;
+
+export type CategorizerPromptJSON = {
+    type: WidgetType;
```

Comment:
I think this should just be `type: "categorizer"` as it really shouldn't be any other type for this structure, should it?